cookbook/deployment/workflow/lp_schedules.py

```diff
@@ -45,8 +50,78 @@ def date_formatter_wf(kickoff_time: datetime):
 )
 
 # %%
-# Fixed Rate
-# ==========
+# This ``schedule`` object can then be used in the construction of a :py:class:`flytekit:flytekit.LaunchPlan`
+# 
+# Complete Cron Example
+# ----------------------
+
+# For example, take the following workflow:
+
+.. code:: python
+
+    from flytekit workflow
+
+    @workflow
+    def MyWorkflow(an_input: int, another_input: int=10):
+        ....
+
+# %%
+# The above workflow can be run on a cron schedule every 5 minutes like so:
+
+.. code:: python
+
+    from flytekit import CronSchedule, LaunchPlan
+
+    cron_lp = LaunchPlan.create(
+        "my_cron_lp",
+        MyWorkflow,
+        schedule=CronSchedule(cron_expression="0 5 * * ? *"),
+        fixed_inputs={"an_input": 5},
+    )
```

Comment:
These lines can be removed.

Reply:
```suggestion
The `kickoff_time_input_arg` corresponds to the workflow input `kickoff_time`. This means that the workflow gets triggered only after the specified kickoff time, and it thereby runs at 10 AM UTC every day.
```

Reply:
The ``kickoff_time_input_arg`` corresponds to the workflow input ``kickoff_time``. This means that when the workflow gets triggered at the specified kickoff time, the kickoff time itself of 10 AM UTC is passed into the workflow's ``kickoff_time`` input.

Reply:
So leave lines 53-80 or remove them?

Reply:
Can you please remove them?